test/codecs/CirceDecodersTest.scala

```diff
@@ -21,76 +21,6 @@ import io.circe.generic.auto._
 
 class CirceDecodersTest extends WordSpec with MustMatchers {
 
-  "referrerAcquisitionDataCodec" should {
```

Comment:
were these tests supposed to be added in the other pr https://github.com/guardian/support-libraries/pull/6/files ?

Reply:
They're not needed any more, there are already tests covering this serialisation in support-models